cms/static/js/spec/views/previous_video_upload_list_spec.js

```diff
@@ -1,29 +1,81 @@
 define(
-    ['jquery', 'underscore', 'backbone', 'js/views/previous_video_upload_list', 'common/js/spec_helpers/template_helpers'],
-    function($, _, Backbone, PreviousVideoUploadListView, TemplateHelpers) {
+    ['jquery', 'underscore', 'backbone', 'js/views/previous_video_upload_list',
+     'common/js/spec_helpers/template_helpers', 'edx-ui-toolkit/js/utils/spec-helpers/ajax-helpers'],
+    function($, _, Backbone, PreviousVideoUploadListView, TemplateHelpers, AjaxHelpers) {
         'use strict';
         describe('PreviousVideoUploadListView', function() {
+            var videoHandlerUrl = '/videos/course-v1:org.0+course_0+Run_0',
+                render = function(numModels) {
+                    var modelData = {
+                        client_video_id: 'foo.mp4',
+                        duration: 42,
+                        created: '2014-11-25T23:13:05',
+                        edx_video_id: 'dummy_id',
+                        status: 'uploading'
+                    };
+                    var collection = new Backbone.Collection(
+                        _.map(
+                            _.range(numModels),
+                            function(num, index) {
+                                return new Backbone.Model(
+                                    _.extend({}, modelData, {edx_video_id: 'dummy_id_' + index})
+                                );
+                            }
+                        )
+                    );
+                    var view = new PreviousVideoUploadListView({
+                        collection: collection,
+                        videoHandlerUrl: videoHandlerUrl
+                    });
+                    return view.render().$el;
+                },
+                verifyVideosInfo;
+
             beforeEach(function() {
-                TemplateHelpers.installTemplate('previous-video-upload', true);
+                setFixtures('<div id="page-prompt"></div>');
+                TemplateHelpers.installTemplate('previous-video-upload');
                 TemplateHelpers.installTemplate('previous-video-upload-list');
             });
 
-            var render = function(numModels) {
-                var modelData = {
-                    client_video_id: 'foo.mp4',
-                    duration: 42,
-                    created: '2014-11-25T23:13:05',
-                    edx_video_id: 'dummy_id',
-                    status: 'uploading'
-                };
-                var collection = new Backbone.Collection(
-                    _.map(
-                        _.range(numModels),
-                        function() { return new Backbone.Model(modelData); }
-                    )
-                );
-                var view = new PreviousVideoUploadListView({collection: collection});
-                return view.render().$el;
+            verifyVideosInfo = function(test, confirmRemove) {
```

Comment:
This is the reference to the test which called this method. We are using `test` at other specs also.